tests/plugins/testsqliteplugin/testsqliteplugin.pro

```diff
@@ -16,5 +16,5 @@ HEADERS += \
 SOURCES += \
     $$PWD/../../../plugins/sqliteplugin/plugin.cpp
 
-target.path=/usr/lib/sailfish/secrets/
+target.path=/usr/lib/Sailfish/Secrets/
```

Comment:
Not a biggie, but changing installation paths in documentation commit is a bit funny.